src/commands/server/start.ts

```diff
@@ -141,6 +152,18 @@ export default class Start extends Command {
       this.error(`🛑 Current platform is ${flags.platform }. Minishift addon is only available on top of Minishift platform.`)
     } else if (flags.platform === 'minishift' && flags.installer && flags.installer === 'helm') {
       this.error(`🛑 Current platform is ${flags.platform }. Helm installer is only available on top of Minikube platform.`)
+    } else if (flags.platform === 'openshift') {
+      if (flags.installer) {
+        if (flags.installer === 'helm') {
+          this.error(`🛑 Current platform is ${flags.platform }. Helm installer is only available on top of Minikube platform.`)
```

Comment:
"Helm installers is only available on top of Minikube platform should be changed" in "...on top of Minikube and Kubernetes platforms"

Reply:
fixed

Reply:
I've changed the logic to be based on the installer instead of the platform